portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -106,9 +106,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<BlogMessage> BlogMessages = getBlogsFromDatastore();
 
     // Create BlogHashMap Object and put BlogMessages in the map.
+    
+    List<BlogMessage> BlogMessagesReplies = new ArrayList<BlogMessage>();
+    for (BlogMessage message : BlogMessages) {
+      if (message.getParentID() != 0) {
+      BlogMessagesReplies.add(message);
+      BlogMessages.remove(message);
+      }
+    }
+
+    for (BlogMessage post : BlogMessages) {
+        long postTime = post.getTimestamp();
+        for (BlogMessage reply : BlogMessagesReplies) {
+          if (reply.getParentID() == postTime) {
+              post.setAdditionalReply(reply);
+          }
+        }
+    }
+
+   // TAYYABA BRANCH
```

Comment:
This comment may be confusing to readers. Can we just remove it? I think we should keep the master branch always consistent.

Reply:
these are old commits, the most recent ones do not have this!(: